dcc-storage-client/src/test/java/org/icgc/dcc/storage/client/ClientMainTest.java

```diff
@@ -86,12 +86,17 @@ public void testMainDownloadWithNonExistentManifest() throws Exception {
   @Test
   public void testMainDownloadWithNonExistentOutDir() throws Exception {
     val file = tmp.newFile();
-    val outDir = new File("/foo");
+    val outDir = new File("/tmp/foo");
```

Comment:
Should probably  use `@Rule TemporaryDir temp` for this. The other tests don't need this because they are read-only, but if the assertion fails below, you will have an invalid stating state assumption.